src/storage/common_storage.ts

```diff
@@ -32,6 +32,8 @@ import { createGeneratorContext } from '../editor/generator_context';
 
 // Types, constants, and functions related to modules, regardless of where the modules are stored.
 
+export const CURRENT_VERSION = '0.1';
```

Comment:
I wonder if we should use the blockly version instead of having a version specific for the file format